user_guide_src/source/helpers/form_helper.rst

```diff
@@ -510,3 +510,58 @@ The following functions are available:
         <input type="radio" name="myradio" value="1" <?= set_radio('myradio', '1', true) ?> />
         <input type="radio" name="myradio" value="2" <?= set_radio('myradio', '2') ?> />
 
+.. php:function:: validation_errors()
+
+    :returns:   The validation errors
+    :rtype:    array
+
+    This function was introduced in v4.3.0.
```

Comment:
This is good for now but it might be good moving forward to hard a "Since:" field to our User Guide class doc template.

Reply:
What's our User Guide class doc template?

Reply:
Sorry, I just meant: we should add this an official info field, following `:rtype:`. On closer inspection of Sphinx though it looks like this may not be possible: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists